packages/cozy-harvest-lib/test/helpers/konnectors.spec.js

```diff
@@ -198,6 +198,7 @@ describe('Konnectors Helpers', () => {
       'USER_ACTION_NEEDED.ACCOUNT_REMOVED',
       'USER_ACTION_NEEDED.CHANGE_PASSWORD',
       'USER_ACTION_NEEDED.PERMISSIONS_CHANGED',
+      'USER_ACTION_NEEDED.SCA_REQUIRED',
```

Comment:
No need to test USER_ACTION_NEEDED.WEBAUTH_REQUIRED?

Reply:
I am actually a bit dubious about this test, since we use a data structure here to list all the errors, I am wondering if it is necessary to test the logic to have all the errors.